portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,129 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+        <header>
+            <nav class="nav">
+                <ul class="nav_list">
+                    <li class="nav_item"><a href="#about" class="nav_link">About Me</a></li>
+                    <li class="nav_item"><a href="#org" class="nav_link">Organisation</a></li>
+                    <li class="nav_item"><a href="#skills" class="nav_link">Skills</a></li>
+                    <li class="nav_item"><a href="#facts" class="nav_link">Random Facts</a></li>
+                    <li class="nav_item"><a href="#gallery" class="nav_link">Gallery</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <!-- About Me -->
+        <section class="about-me" id="about">
+            <img src="/img/aastha01.jpg" alt="Aastha smiling" class="about_img">
+            <h2 class="section_title section_title-about">Hi,I am <strong>Aastha</strong></h2>
+            <p class="section_subtitle section_subtitle-about">Undergraduate student at Nirma University, Ahmedabad.</p>
+            
+            <div class="about-me_body">
+                <p>I enjoy turning complex problems into simple, beautiful and intuitive codes. When I'm not coding or solving puzzles, you'll find me creating art stuff, doing social work or sleeping :)</p>
+            </div>          
+            
+         </section>
+
+         <!-- Organisation-->
+         <section class="org" id="org">
+            <h2 class="section_title section_title-org">Organisation</h2>
+            <div class="org">
+                <table width="100%">
+                    <tr>
+                        <th><h3>CodeAdda</h3></th>
+                        <th><h3>Prarambh</h3></th>
+                        <th><h3>NSS</h3></th>
+                    </tr>
+                    <tr>
+                        <td>Club Director</td>
+                        <td>Content writer</td>
+                        <td>Teaching Volunteer</td>
+                    </tr>
+                    <tr>
+                        <td>(09/19 - present)</td>
+                        <td>(05/20 - present)</td>
+                        <td>(07/18 - present)</td>
+                    </tr>
+                    <tr>
+                        <td><br>Programming club, Nirma University: Organizing sessions on advanced Dat Structures and algorithms and contests on competitive platforms to spread coding culture.</td>
+                        <td><br>ITNU alumni engagement and student mentoring platform: Alumni share their experience and learnings and guide the students throughout their college-to-carrier journey.</td>
```

Comment:
The width of these lines is too long. Can you break them into multiple lines?

Reply:
Sure, will do.

Reply:
Done, plz review.